### PR TITLE
Add subscription platform emails

### PR DESCRIPTION
**Pull request by clouserw**

### CODE REVIEW

**Comment by jklukas** [comment]
The spell-check task is failing here, probably because `Stripe` (capitalized) needs to be added to [the](https://github.com/mozilla/data-docs/blob/main/.spelling)

**Comment by clouserw** [comment]
Thanks.  I added it but it's still broken.  Is there a place to see more verbose output from the spellchecker?

**Comment by badboy** [comment]
> Thanks. I added it but it's still broken. Is there a place to see more verbose output from the spellchecker?

You can run spellchecking locally using `script/spell_check.sh`, which will give you colored output:

image

Unfortunately the color is lost on GhA and there's no way to tell mdspell to disable color and use some other output.

tl;dr: add `webhook` to `.spelling` and you're all good.

**Comment by clouserw** [comment]
Thanks @badboy .  It really helps to have the color!

**Comment by jklukas** [comment]
Thanks for this updates. Sorry that the review cycle took so long.
